sites/default/statement.inc.php

```diff
@@ -442,6 +447,9 @@ function create_statement($stmt) {
  #minimum_amount_to _print
  if ($stmt[amount] <= ($GLOBALS['minimum_amount_to_print']) && $GLOBALS['use_statement_print_exclusion']) return "";
 
+ // Don't print if decreased
+ if ($GLOBALS['disallow_print_deceased']) return "";
```

Comment:
if you are printing 100 statements do you want a statement printed with "Your patient is DEAD. You must start a new game or load a saved one!" stuck some where in it?